### DISCUSSION TITLE
Attention Self Hosters: Breaking Configuration Changes

### DISCUSSION THREAD

**Discussion by dyc3** [dyc3/opentogethertube#853] [Announcements]
In #849, I'm making major changes to how OTT is configured. I've done my best to make it backwards compatible, but there _is_ a chance that it could break your deployments if you have an auto updater.

To smooth over this change as much as possible, if you encounter an issue when updating **please file an issue**. While your issue is getting resolved, you can revert your deployment to the [`v0.3.0` docker tag](https://hub.docker.com/layers/dyc3/opentogethertube/v0.3.0/images/sha256-e4d3e05e8bf1f8f7ab4cdf64ea53cff8702a59cd57c8ad702e994e632a5e20fc?context=repo) (or the git tag with the same name).

At the absolute earliest, I'll be merging #849 on April 22, 2023 at noon EST. The docker tag is already available, so if you want to defer the problem until later, switch to that tag.

**Comment by gianklug** [comment]
My watchtower just updated it automatically, seems to have broken some things. Will investigate later and comment what I had to fix.

**Reply by dyc3** [reply]
Any updates? If you had problems you're likely not the only one, and I'd like to deploy any fixes ASAP.

**Comment by gianklug** [comment]
Sorry for forgetting this.

Somehow I had the DB mode still set to sqlite in the old production.env.
OTT crashed because of a missing Sqlite module.

After removing that line and adding the new env files it's working again.

Carson McManus ***@***.***> schrieb am Do., 4. Mai 2023,
15:33:
